app/views/catalog/_group.html.erb

```diff
@@ -0,0 +1,27 @@
+<% # container for all groups in index view -%>
```

Comment:
A basic port over from Blacklight https://github.com/projectblacklight/blacklight/blob/master/app/views/catalog/_group.html.erb

This adds some enhancement but likely will be changed as we implement additional designed tickets.